Text/RegexExtractText/pom.xml

```diff
@@ -51,7 +51,7 @@
     <dependency>
       <groupId>org.json</groupId>
       <artifactId>json</artifactId>
-      <version>20210307</version>
+      <version>20230227</version>
```

Comment:
Would it be better to use:
```
  <version>[20230227)</version>
```
as suggested in:
https://github.com/datalogics/apdfl-java-maven-samples/security/dependabot/2
?
Seems like there could be downsides, particularly if we don't run this on a regular basis, but it should be considered.

Reply:
I almost did that, and with that nudge I will.

Reply:
Fixed